### ISSUE TITLE
Use quadtrees for pin clustering

### ISSUE THREAD

**Issue by vinnyoodles** [QSport/QSport#172]

**Comment by rishipulluri** [comment]
Hello, I respectfully disagree

**Comment by vinnyoodles** [comment]
@rishipulluri why

**Comment by rishipulluri** [comment]
cuz we have to implement one

**Comment by vinnyoodles** [comment]
That's not a reason to not do it.

**Comment by seanpc9** [comment]
I agree with Vincent. If this thing it going to hit the market, we have to embrace challenges, not run away from them.

**Comment by rishipulluri** [comment]
alright, you convinced me.

**Comment by rishipulluri** [comment]
actually my bad, we definately actually need a quadtree for the pin
display. There is no way our current display is good enough. Lesson learnt,
never speak before researching.
